<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginGui</class>
 <widget class="QWidget" name="PluginGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QRadioButton" name="affineRadio">
     <property name="text">
      <string>Affine</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Interpolation :</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Border Type :</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="interpolationCombo"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QRadioButton" name="resizeHalfRadio">
     <property name="text">
      <string>Resize (1/2 X)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QRadioButton" name="resizeDoubleRadio">
     <property name="text">
      <string>Resize (2 X)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QRadioButton" name="remapRadio">
     <property name="text">
      <string>Remap</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="borderTypeCombo"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QRadioButton" name="perspectiveRadio">
     <property name="text">
      <string>Perspective</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
